etcdctl/ctlv3/command/migrate_command.go

```diff
@@ -311,6 +313,10 @@ func defaultTransformer() (io.WriteCloser, io.ReadCloser, chan error) {
 				return
 			}
 
+			if migrateExcludeTTLKey && node.TTL != 0 {
```

Comment:
we should move this to writeKey func, which emits the keys to transformer. or this flag only works for the default transformer.